forgettable-frontend/src/components/CompanyCard/CompanyCard.js

```diff
@@ -0,0 +1,156 @@
+import {Avatar, AvatarGroup, StyledEngineProvider, SvgIcon} from '@mui/material';
+import React from 'react';
+import classes from './CompanyCard.module.css';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
+import {convertSocialMediaToIcon} from '../../functions/socialMediaIconConverter';
+import PropTypes from 'prop-types';
+import {getFirstMetTimeString, getLongDateStringWithSlashes} from '../../functions/dateFormatter';
+import UnknownDetail from '../UnknownDetail/UnknownDetail';
+import {getImageSrcFromBuffer} from '../../functions/getImageSrcFromBuffer';
+
+/*
+ * Component for displaying information of a company in a list.
+ * This is the full version of the company card.
+ *
+ * Author: Andy Kweon (skwe902)
+ */
+const CompanyCard = (props) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
```

Comment:
Following on from previous comment, if changed, this becames React.useState(null) -> useState(null)